packages/fxa-settings/package.json

```diff
@@ -237,6 +238,7 @@
     "grunt-cli": "^1.4.3",
     "grunt-contrib-concat": "^2.1.0",
     "grunt-contrib-watch": "^1.1.0",
+    "grunt-hash": "^0.5.0",
```

Comment:
I'm sure you saw this but just noting that this was last published 10 years ago, gets 8k downloads a week, and was archived. However I'm not sure if I see a better option right away seeing as we're using grunt for other related tasks and other grunt packages that do hashing seem to be just as old. It'd be nice if webpack could handle it for us. 😩

Reply:
FWIW, this is what I tried to do first, but it didn't go smoothly, and I bailed because of time constraints. I'd be all for another pass where we use webpack though. The issue is that to do this with webpack, it seems like the l10n files be be packed into the bundle. However, that means we should resolve them with imports, and that was a big enough departure that I switched to using grunt. Happy to file a follow up for this though. We also have some assets that would be better resolved as imports, but again probably outside the scope of this PR.

Reply:
Yeah, that's exactly what I figured. I vote for a follow up and throwing it into React Cleanup.